pipeline/genomeBrowserTrack/brcaExchange.as

```diff
@@ -0,0 +1,142 @@
+table brcaExchanges
+" bed9 + many additional fields from brcaExchange.org "
+    (
+    string chrom;      "Chromosome (or contig, scaffold, etc.)"
```

Comment:
it's likely that fields will change in the future -- is there a way to make this file update accordingly or will manual changes be required here?